samples/videoDecodeRGB/videodecrgb.cpp

```diff
@@ -276,13 +276,15 @@ int main(int argc, char **argv) {
 
         std::string device_name, gcn_arch_name;
         int pci_bus_id, pci_domain_id, pci_device_id;
-        hipStream_t stream = viddec.GetStream();
 
         viddec.GetDeviceinfo(device_name, gcn_arch_name, pci_bus_id, pci_domain_id, pci_device_id);
         std::cout << "info: Using GPU device " << device_id << " " << device_name << "[" << gcn_arch_name << "] on PCI bus " <<
         std::setfill('0') << std::setw(2) << std::right << std::hex << pci_bus_id << ":" << std::setfill('0') << std::setw(2) <<
         std::right << std::hex << pci_domain_id << "." << pci_device_id << std::dec << std::endl;
         std::cout << "info: decoding started, please wait!" << std::endl;
+        // Jefftest
+        HIP_API_CALL(hipStreamCreate(&hip_stream_dec));
```

Comment:
I don't really think we need to 2 streams here as we can do cc kernel only after the copy is finished. So there is no advantage. Plus the earlier implementation was also using the same stream object for both. @AryanSalmanpour : please confirm.